packages/engine/README.md

```diff
@@ -404,7 +404,7 @@ The packages utilize a [communication implementation](./lib/execution/src/runner
 
 #### Stateful
 
-The [`stateful` crate](./lib/stateful) is the backend responsible for keeping the data between the simulation run main loops and language runners in sync. It encapsulates logic surrounding read/write access, as well as low-level shared memory representation.
+The [`stateful` crate](./lib/stateful) is responsible for keeping the data in the main simulation loops in sync with the language runners. It encapsulates logic surrounding read/write access, as well as low-level shared memory representation.
```

Comment:
Upon review, not actually sure this is correct. We talk about the 'main loop' as a concept (sometimes proper cased in our materials). "main simulation loops" may not necessarily imply the same thing. @Alfred-Mountfield or @TimDiekmann to confirm.

Reply:
`stateful` uses the `memory` crate, which is able to store data in memory. It provides data structures, layouts, and access. It is *not* aware of the simulation loop but is used *by* the simulation loop to keep the data in sync. I see both versions possibly misleading. Could you expand on this section a little bit? The crate-level module describes the `stateful` crate pretty well I think.

Reply:
> handles the sharing of memory between different runners.

This still sounds like the `stateful` crate is actively handling the sharing, but the crate is unaware of the execution crate.

Reply:
Sorry for the wasted time in discussing this, this section is being removed as part of #410 so we don't have to redefine the description of crates in multiple places. The source of truth should be their crate docs.